content/posts/wop.md

```diff
@@ -0,0 +1,417 @@
+- has functional feedback and following mechanisms for readers
+- lets readers opt-in to sharing their email with you, which you can integrate directly with
+  Mailchimp if you want
+- lets you notify readers when you've updated your book, if you want (separate from mailing list)
+- allows you to split your revenue amongst your collaborators automatically---you don't need to hold
+  cash and do payouts, LeanPub will handle that for you
+- enables you to automatically direct some or all your revenue to a charitable organization
+- has a functional newsletter you can spend revenue to secure a spot in (they also highlight
+  projects that _don't_ spend money)
+- lets you sell multiple packages of your project at varying prices (e.g. just the text, text +
+  assets, text + assets + consulting/custom content)
+- enables you to set a minimum price and a suggested price and uses a price slider that _starts_ at
+  the suggested price (my experience is that many people actually pay _more_ than the suggested
+  price)
+- supports coupons (which you can use to mimic community copies, if you want)
+- has robust settings including:
+  - a community edition (free copy with optionally limited content that people get in exchange for
+    opting to provide their email address)
+  - a sample edition (free copy with optionally limited content, no conditions)
+  - the ability to temporarily unlist your book
+  - provide your book in multiple languages
+  - incorporate team bios
+  - set a thank you message for purchases
+
+For me, the most useful features are the automatic revenue sharing with the team and being able to
+automatically direct revenue to a charity organization. I find that this is worth the 20% cut even
+before considering the rest of the features and benefits.
+
+When it comes to pricing, I'm comfortable selling my work on LeanPub at similar rates to those
+people are charging for physical copies. I set the suggested price where I think it ought to be and
+then fiddle with my minimum price (usually in the $8--12 range). One model I really like on LeanPub
+is to start the minimum price lower when I first release and increase it as my team adds assets and
+improves the core text. For example, I might put a book up with a $32 suggested price and raise the
+minimum $1 every month that we add new stuff until the minimum is $16.
+
+Remember, you can have add-ons and packages and price those separately, too.
+
+## Physical
+
+I approach physical editions a little differently. I may not do a physical edition for every
+project---it depends on how much interest and excitement there is in both the team and the audience.
+But when I do, I aim for a luxury option. Because I release a free web edition and an affordable
+offline edition, I don't worry about how affordable or limited my physical editions may be. Instead
+I can work with the team to manifest the best possible expression of our work in physical form.

```

Comment:
```suggestion
When I do, I aim for a luxury option. Because I release a free web edition and an affordable offline
edition, I don't worry about how affordable or limited my physical editions may be. Instead I can
work with the team to manifest the best possible expression of our work in physical form.
```